src/ray/core_worker/core_worker.cc

```diff
@@ -2547,4 +2553,8 @@ void CoreWorker::SetActorTitle(const std::string &title) {
   actor_title_ = title;
 }
 
+std::string CoreWorker::GetSerializedJobConfig() const {
```

Comment:
Why not return `job_config_` directly and let the caller decide whether to serialize it or not?

Reply:
Okey, I will fix it.